### ISSUE TITLE
Setting up translation path for Lang to get right strings

### ISSUE THREAD

**Issue by MarcinOrlowski** [orchestral/testbench#167]
Hi,

I am using your package to test my Laravel module and all works fine but Laravel's `Lang`. I suspect it's simply because it expects the files to reside in `storage/lang/<locale>/<file>` while in my current source tree it's just `lang/<locale>/<file>` and because of that I am getting either `null` (when I call `Lang::get()` from my unit test method) or the lang key (i.e. `namespace.unknown_error`) when I call `Lang::get()` from method being tested. What is the proper way to make `Lang` work in this env as I am currently banging my head with no progress. Thanks in advance.

**Comment by crynobone** [comment]
Pretty sure this is an implementation problem, Testbench doesn't mess around with Laravel internals, we just help to boot-up Laravel application for package development.

If you think this is an actual bug, submit a failing test on this repository so I can fix it.

**Comment by MarcinOrlowski** [comment]
Seems I just needed a sleep :) as I just figured what was the culprit. As usual it's pretty obvious once you find out, but when you are stuck then even obvious things come up harder so hopefully this comment would help others (or perhaps it'd be worth editing docs with such note).

The package I develop use `ServiceProvider` subclass and in my provider's `boot()` method I set up the runtime environment, including paths to localization strings or config file path. I use `__DIR__` constant there to construct full paths and this const is different when I tests. In result all the paths were pointing into wrong location.

As solution I created `TestServiceProvider` that extends package's `ServiceProvider` and overrides its `boot()` method. I do call `parent::boot();` to do all the heavy lifting first and then I set paths again so it takes `__DIR__` difference into account (for localization I call usual `$this->loadTranslationsFrom()`). 

Finally, my base test class's `getPackageProviders()` return newly created `TestServiceProvider` and all seems to work fine now.
